src/Panel/Panel.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) "../less/index";
+
+.Panel--container {
+  border-style: solid;
+  border-width: @size_3xs @size_3xs @size_3xs @size_s;
+  border-radius: @size_2xs;
+  padding: @size_xs;
```

Comment:
Can use spacing mixins here, and all the places you directly reference the sizing variables

e.g.:  .padding--xs